### PR TITLE
infra/labeler: Fix pull request labeler workflow

### PR DESCRIPTION
**Pull request by gabrielmocanu**
- Switch to another GitHub action since the other didn't work with crontab / pull_request
- Update configuration file
- Fix add-labels script

### CODE REVIEW

**Comment by gabrielmocanu** [comment]
You can see the tests that I made here: https://github.com/gabrielmocanu/operating-systems/pull/5

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -21,5 +22,5 @@ jobs:
       - uses: actions/github-script@v6
         with:
           script: |
-            const { default: addLabelsToPR } = await import('${{ github.workspace }}/.github/scripts/add-labels.js') ;
-            await addLabelsToPR({github, context})
+            const { default: doLabels } = await import('${{ github.workspace }}/.github/scripts/add-labels.js') ;
+            await doLabels({github, context})
```
Comment (teodutu) [inline .github/workflows/labeler.yml:26]:
```suggestion
            const { default: assignLabelsToPRs } = await import('${{ github.workspace }}/.github/scripts/add-labels.js') ;
            await assignLabelsToPRs({github, context})
```

File: .github/scripts/add-labels.js [hunk 2]
```diff
@@ -29,29 +29,30 @@ async function createLabelsForPR(github, context, pr) {
...
+    await github.request('POST /repos/{owner}/{repo}/issues/{issue_number}/labels', {
         owner: context.repo.owner,
         repo: context.repo.repo,
         issue_number: pr,
         labels: labels
     })
 }
 
-export default async ({github, context}) => {
+export default async function doLabels({github, context})
```
Comment (teodutu) [inline .github/scripts/add-labels.js:45]:
```suggestion
export default async function assignLabelsToPRs({github, context})
```

**Review by teodutu — CHANGES_REQUESTED** [review]
`doLabels` is too vague of a name. Why not `assignLabelsToPRs`? Other than this, I have faith (and nothing else) that this time it'll work.

**Review by teodutu — APPROVED** [review]
Godspeed! ☦

**Comment by gabrielmocanu** [comment]
@Alex-deVis  Please approve it and merge it if it's OK.

**Pull request merged**